.changeset/new-peas-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
```

Comment:
Hey @loouis-t could you change "minor" to "patch" ?

Reply:
Thank you @loouis-t 
Could you also rebase ?

Reply:
Done!